WNPRC_Purchasing/build.gradle

```diff
@@ -0,0 +1,20 @@
+import org.labkey.gradle.util.BuildUtils
+
+dependencies {
+    implementation "net.sf.opencsv:opencsv:${opencsvVersion}"
```

Comment:
I don't think any of these four dependencies are relevant at this point, correct?

Reply:
correct, removed